code/modules/vending/engineering.dm

```diff
@@ -11,7 +11,7 @@
 		            /obj/item/clothing/head/hardhat = 4,
 					/obj/item/storage/belt/utility = 4,
 					/obj/item/clothing/glasses/meson/engine = 4,
-					/obj/item/clothing/gloves/color/yellow = 4,
+					/obj/item/clothing/gloves/color/yellow = 7,
```

Comment:
this is the special engi vendor you can only get from cargo that nobody ever gets because its a pain in the ass

Reply:
ye i now have changed the you tool one too